ios/brave-ios/Sources/Brave/WebFilters/ShieldStats/Adblock/AdBlockStats.swift

```diff
@@ -143,6 +130,12 @@ public actor AdBlockStats {
       return
     }
     self.resourcesInfo = resourcesInfo
+
+    if #available(iOS 16.0, *) {
+      ContentBlockerManager.log.debug(
+        "Updated resources component: `\(resourcesInfo.localFileURL.path(percentEncoded: false))`"
+      )
+    }
```

Comment:
You probably want to log this regardless and just use deprecated `path` property)

Reply:
to be honest I did this because i like the format better and I don't have to manually escape spaces when i want to go to that path.